r/tests/testthat/helper-skip.R

```diff
@@ -97,6 +97,7 @@ skip_on_linux_devel <- function() {
   # Skip when the OS is linux + and the R version is development
   # helpful for skipping on Valgrind, and the sanitizer checks (clang + gcc) on cran
   if (on_linux_dev()) {
+    return()
     skip_on_cran()
```

Comment:
Can we remove this?

Reply:
This was me trying to trigger ubsan failures on linux that we were seeing on CRAN's M1, but that didn't work (I'll back those changes out). I _have_ found a way I believe to run the M1 checks, which I'm trying right now. Once I get a failure there I'll push the fixes I have too (but I want to prove to myself that our CI is catching this before pushing the proposed fixes). I'll tag you when this is ready, as always thanks for your help 🙏

Reply:
Oh, sorry!